### ISSUE TITLE
Regenerate test jobs to Add BUILD_TYPE

### ISSUE THREAD

**Issue by AdamBrousseau** [eclipse/openj9#5210]
I have created views with regexes to help sort the new separated jobs.
Please regenerate all the Test jobs to include the build type suffix
1. Nightly
1. OMR
1. Personal
1. Release

ex. Test-sanity.functional-JDK8-linux_x86-64_cmprssptrs_Nightly

Related #5182 
cc @llxia

**Comment by llxia** [comment]
Related to https://github.com/AdoptOpenJDK/openjdk-tests/issues/1037

**Comment by AdamBrousseau** [comment]
@llxia are we waiting on 1037 to be closed before we complete this or will we open a another issue when the job names change?

**Comment by llxia** [comment]
We will go with the naming convention that is proposed in 1037. I will update the code to regenerate the jobs.

**Comment by AdamBrousseau** [comment]
Thanks Lan, I'll add to my PR to update with the new test job names.

**Comment by llxia** [comment]
We may need to create a SPEC to ARCH_OS mapping.

**Comment by llxia** [comment]
Test jobs are generated for above 4 suffixes for JDK8 and JDK11 in OpenJ9 Jenkins.

Below is the SPEC to ARCH_OS map that is needed in `get_test_job_name()`. We may also want to add map for `aarch32_linux` and `aarch64_linux` if we are going to run them in OpenJ9 soon.

```
def SPEC_ARCH_OS_MAP = [ 
	'aix_ppc-64_cmprssptrs' : ppc64_aix',
	'linux_ppc-64_cmprssptrs_le' : 'ppc64le_linux',
	'linux_390-64_cmprssptrs' : 's390x_linux',
	'zos_390-64_cmprssptrs' : 's390x_zos',
	'linux_x86-64_cmprssptrs' : 'x86-64_linux',
	'linux_x86-64' : 'x86-64_linux_xl',
	'osx_x86-64_cmprssptrs' : 'x86-64_mac',
	'osx_x86-64' : 'x86-64_mac_xl',
	'win_x86-64_cmprssptrs' : 'x86-64_windows',
	'win_x86' : 'x86-32_windows'
]
```

**Comment by AdamBrousseau** [comment]
Great. We will also need JDK12,13 and 'next' which might need another conversation.

**Comment by llxia** [comment]
JDK12,13 and Next jobs are generated.

**Comment by AdamBrousseau** [comment]
I believe this is complete
